src/commondao/common.dao.ts

```diff
@@ -666,6 +679,7 @@ export class CommonDao<
       this.assignIdCreatedUpdated(dbm, opt) // mutates
       dbm = this.anyToDBM(dbm, opt)
       if (opt.ensureUniqueId && idWasGenerated) await this.ensureUniqueId(table, dbm)
+      if (this.cfg.immutable === true) this.requireObjectMutability()
```

Comment:
Ok, here I miss "loading" to check if the object actually exists or not. Looks like it'll always throw there (if immutable == true).